requirements-dev.txt

```diff
@@ -18,3 +18,5 @@ pytest-faulthandler<2.0; platform_python_implementation == "PyPy"
 pytest-timeout
 feedparser
 python-hglib
+flake8
+
```

Comment:
Small thing, but better to not introduce arbitrary blank lines, and keep our changes to exactly what we want.